randovania/games/prime1/json_data/Chozo Ruins.txt

```diff
@@ -1257,15 +1261,22 @@ Extra - asset_id: 1227669322
   * Extra - dock_index: 1
   > Pickup (Missile Expansion)
       All of the following:
-          After Flaahgra or Poisoned Water Damage ≥ 57
           Any of the following:
-              Morph Ball Bomb and Gravity Suit and Morph Ball and Bomb Jump (Beginner)
+              After Flaahgra or Poisoned Water Damage ≥ 57
+              Gravity Suit and Poisoned Water Damage ≥ 25
+          Any of the following:
+              Morph Ball Bomb and Gravity Suit and Morph Ball and Bomb Jump (Beginner) and Poisoned Water Damage ≥ 15
               No Gravity Suit and Slope Jump (Advanced) and Gravityless Underwater Movement (Advanced) and Enabled Allow Dangerous Gravity Suit Logic
               All of the following:
                   Space Jump Boots
                   Any of the following:
                       Gravity Suit
                       Slope Jump (Beginner) and Gravityless Underwater Movement (Beginner)
+              All of the following:
+                  Boost Ball and Gravity Suit and Morph Ball and Wall Boost (Beginner)
+                  Any of the following:
+                      Movement (Intermediate) and Poisoned Water Damage ≥ 65
+                      Movement (Expert) and Poisoned Water Damage ≥ 20
```

Comment:
If I'm reading this right, there is an "All of the following" that groups everything here together, which I feel is a mistake.

Reply:
![image](https://user-images.githubusercontent.com/14116556/149708556-b7c2170a-9d84-4f58-b368-8ab92d92a4f8.png)
This is the 4th of 4 options to get to the item. The first all of the following is just a way of using different damage requirements for walking to the base of the vertical obstacle (you can walk faster underwater w/ grav).

Reply:
Oh that makes sense. Yeah this is fine then.